### PR TITLE
Update do.m3u

### PR DESCRIPTION
**Pull request by Pecaquito**

### CODE REVIEW

File: streams/do.m3u [hunk 1]
```diff
@@ -129,8 +163,6 @@ https://vdo1.streamgato.us:3833/live/canalhtvlivelive.m3u8
 https://imagenuniversaltv.net:3771/live/iutvlive.m3u8
 #EXTINF:-1 tvg-id="JazminTV.do",Jazmín TV (360p)
 https://tvdatta.com:3957/hybrid/play.m3u8
-#EXTINF:-1 tvg-id="JazminTV.do",Jimani TV (360p)
-https://cloud2.streaminglivehd.com:1936/8246/8246/playlist.m3u8
```
Comment (Dum4G) [inline streams/do.m3u:133]: image

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/do.m3u [hunk 2]
```diff
@@ -168,7 +200,11 @@ https://cdn.streamingcpanel.com:3526/live/teleraiceslive.m3u8
 #EXTINF:-1 tvg-id="NexxoTV.do",Nexxo TV (720p) [Not 24/7]
 https://cdn3.streamgato.us:3938/live/nexxotvlive.m3u8
 #EXTINF:-1 tvg-id="NisibonTV.do",Nisibon TV (480p)
+https://ssh101-fl.bozztv.com/ssh101/nisibontv/playlist.m3u8
```
Comment (Dum4G) [inline streams/do.m3u:203]: does not work

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/do.m3u (hunk 2 above)
Comment (Pecaquito) [reply streams/do.m3u:203]: This Chanel isn't 24/7

File: streams/do.m3u (hunk 1 above)
Comment (Pecaquito) [reply streams/do.m3u:133]: Ok Edited

File: streams/do.m3u [hunk 3]
```diff
@@ -129,8 +163,6 @@ https://vdo1.streamgato.us:3833/live/canalhtvlivelive.m3u8
 https://imagenuniversaltv.net:3771/live/iutvlive.m3u8
 #EXTINF:-1 tvg-id="JazminTV.do",Jazmín TV (360p)
 https://tvdatta.com:3957/hybrid/play.m3u8
```
Comment (Dum4G) [inline streams/do.m3u:165]:
```suggestion
https://tvdatta.com:3957/hybrid/play.m3u8
#EXTINF:-1 tvg-id="JazminTV.do",Jimani TV (360p)
https://cloud2.streaminglivehd.com:1936/8246/8246/playlist.m3u8
```

File: streams/do.m3u [hunk 4]
```diff
@@ -168,7 +200,11 @@ https://cdn.streamingcpanel.com:3526/live/teleraiceslive.m3u8
 #EXTINF:-1 tvg-id="NexxoTV.do",Nexxo TV (720p) [Not 24/7]
 https://cdn3.streamgato.us:3938/live/nexxotvlive.m3u8
 #EXTINF:-1 tvg-id="NisibonTV.do",Nisibon TV (480p)
```
Comment (Dum4G) [inline streams/do.m3u:202]:
```suggestion
#EXTINF:-1 tvg-id="NisibonTV.do",Nisibon TV (480p) [Not 24/7]
```

File: streams/do.m3u [hunk 5]
```diff
@@ -168,7 +200,11 @@ https://cdn.streamingcpanel.com:3526/live/teleraiceslive.m3u8
...
+https://ssh101-fl.bozztv.com/ssh101/nisibontv/playlist.m3u8
+#EXTINF:-1 tvg-id="NisibonTV.do",Nisibon TV
```
Comment (Dum4G) [inline streams/do.m3u:204]:
```suggestion
#EXTINF:-1 tvg-id="NitidaTV.do",Nitida TV
```

**Review by Dum4G — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**